### PR TITLE
feat(pci.storages.databases): datatr-366 - split databases by type

### PR DESCRIPTION
**Pull request by jperchoc**
datatr-366

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         |  no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DATATR-366
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Split databases by types in menu

### :flags: Translations

- [x] CMT-4816

### CODE REVIEW

File: packages/manager/apps/container/src/public/translations/sidebar/Messages_fr_FR.json [hunk 1]
```diff
@@ -86,7 +87,9 @@
   "sidebar_netapp": "Enterprise File Storage",
   "sidebar_cda": "Cloud Disk Array",
   "sidebar_veeamcc": "Veeam Cloud Connect",
-  "sidebar_databases": "Bases de données",
+  "sidebar_pci_analytics_databases": "Databases",
+  "sidebar_pci_analytics_datastreaming": "Data Streaming",
+  "sidebar_pci_analytics_dataanalysis": "Data Analysis",
```
Comment (sachinrameshn) [inline packages/manager/apps/container/src/public/translations/sidebar/Messages_fr_FR.json:92]:
```suggestion
  "sidebar_pci_analytics_data_streaming": "Data Streaming",
  "sidebar_pci_analytics_data_analysis": "Data Analysis",
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/databases.constants.js [hunk 2]
```diff
@@ -138,6 +138,50 @@ export const SSL_MODE_REQUIRED = ['require', 'required', 'REQUIRED'];
...
+      DATABASE_TYPES.MONGO_DB,
+      DATABASE_TYPES.POSTGRESQL,
+      DATABASE_TYPES.MYSQL,
+      DATABASE_TYPES.REDIS,
+      DATABASE_TYPES.CASSANDRA,
+      DATABASE_TYPES.M3DB,
+      DATABASE_TYPES.M3AGGEGATOR,
+    ],
+  },
+  datastreaming: {
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/storages/databases/databases.constants.js:154]:
```suggestion
  data_streaming: {
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/databases.constants.js [hunk 3]
```diff
@@ -138,6 +138,50 @@ export const SSL_MODE_REQUIRED = ['require', 'required', 'REQUIRED'];
...
+    label: 'datastreaming',
+    engines: [
+      DATABASE_TYPES.KAFKA,
+      DATABASE_TYPES.KAFKA_CONNECT,
+      DATABASE_TYPES.KAFKA_MIRROR_MAKER,
+    ],
+  },
+  dataanalysis: {
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/storages/databases/databases.constants.js:162]:
```suggestion
  data_analysis: {
```

Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/databases.constants.js]: Changed: https://github.com/ovh/manager/pull/10083/commits/a1aaf68adc512e0ec90cd2132255694a35ab790c

File: packages/manager/modules/pci/src/projects/project/storages/databases/databases.constants.js (hunk 2 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/databases.constants.js]: Changed: https://github.com/ovh/manager/pull/10083/commits/a1aaf68adc512e0ec90cd2132255694a35ab790c

File: packages/manager/apps/container/src/public/translations/sidebar/Messages_fr_FR.json (hunk 1 above)
Comment (jperchoc) [reply packages/manager/apps/container/src/public/translations/sidebar/Messages_fr_FR.json]: Changed: https://github.com/ovh/manager/pull/10083/commits/a1aaf68adc512e0ec90cd2132255694a35ab790c

**Review by sachinrameshn — APPROVED** [review]

**Review by abullet33 — DISMISSED** [review]

**Review by abullet33 — APPROVED** [review]

**Review by abullet33 — DISMISSED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**